__tests__/server.test.js

```diff
@@ -38,21 +49,65 @@ describe("/api/topics", () => {
     })
 })
 
+describe("/api/articles", () => {
+    describe("GET", () => {
+        test("GET 200: Responds with an array of all article objects sorted by 'date' by default", () => {
+            return request(app)
+            .get("/api/articles")
+            .expect(200)
+            .then(({body}) => {
+                console.log(JSON.stringify(body))
+                expect(body.articles).toHaveLength(13)
+                expect(body.articles).toBeSortedBy("created_at", {descending: true})
+            })
+        })
+        test("GET 200: Responds with an array of all article objects WITHOUT a body property", () => {
+            return request(app)
+            .get("/api/articles")
+            .expect(200)
+            .then(({body}) => {
+                body.articles.forEach((article) => {
+                    expect(article).not.toHaveProperty("body")
+                })
+            })
+        })
+        test("GET 200: Responds with an array of all article objects with the fields- author, title, article_id, topic, created_at, votes, article_img_url and a comment count for each article", () => {
+            return request(app)
+            .get("/api/articles")
+            .expect(200)
+            .then(({body}) => {
+                expect(body.articles[0].comment_count).toBe("2")
+                expect(body.articles[6].comment_count).toBe("11")
+                body.articles.forEach((article) => {
+                    expect(article).toHaveProperty("author")
+                    expect(article).toHaveProperty("title")
+                    expect(article).toHaveProperty("article_id")
+                    expect(article).toHaveProperty("topic")
+                    expect(article).toHaveProperty("created_at")
+                    expect(article).toHaveProperty("votes")
+                    expect(article).toHaveProperty("article_img_url")
+                    expect(article).toHaveProperty("comment_count")
+                })
```

Comment:
Good. Please also check their value types.

Reply:
I'm on it, cheers :D